Pear/Onboarding/AnswerPromptViewController.swift

```diff
@@ -179,6 +180,7 @@ extension AnswerPromptViewController: UITextViewDelegate {
     }
 
     func textView(_ textView: UITextView, shouldChangeTextIn range: NSRange, replacementText text: String) -> Bool {
+        if text == "\n" { textView.resignFirstResponder() }
```

Comment:
There's [`textFieldShouldReturn`](https://developer.apple.com/documentation/uikit/uitextfielddelegate/1619603-textfieldshouldreturn)

Reply:
This is a textview which doesn't have a similar delegate func